anta/tests/routing/bgp.py

```diff
@@ -722,6 +725,8 @@ class BgpPeer(BaseModel):
             """IPv4 address of a BGP peer."""
             vrf: str = "default"
             """Optional VRF for BGP peer. If not provided, it defaults to `default`."""
+            strict: bool = False
+            """Optional check for BGP peer. Verifies the only mentioned capabilities should be listed, otherwise test should fail."""
```

Comment:
```suggestion
            """Optional check to verifies the only mentioned capabilities should be listed, otherwise test should fail. It defaults to false """
```

Reply:
Updated.